### PR TITLE
feat: support virion 3.0

### PR DESCRIPTION
**Pull request by SOF3**

### CODE REVIEW

**Comment by SOF3** [comment]
Hi! After merging this pull request, your users may use Virion 3.0 in their plugins by using `vcs` dependencies in their composer.json, but you could make their lives easier by completing these steps:

1. Tag each release with the `git tag` command or the [GitHub web UI](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
2. Submit your package on Packagist: <https://packagist.org/packages/submit>

Please follow these steps every time you release a new stable version so that user plugins can get their dependencies updated automatically upon the next build.

**Comment by inxomnyaa** [comment]
About time we merge this

**Comment by dktapps** [comment]
This doesn't include a dep for packet hooker, is that intentional?

**Comment by inxomnyaa** [comment]
> This doesn't include a dep for packet hooker, is that intentional?

oops

File: composer.json [hunk 1]
```diff
@@ -4,6 +4,15 @@
     "type": "library",
     "license": "LGPL-3.0-only",
     "autoload": {
-        "psr-4": {"CortexPE\\Commando\\": ["src/CortexPE/Commando/"]}
+        "psr-4": {"CortexPE\\Commando\\": ["src\\CortexPE\\Commando\\"]}
```
Comment (IvanCraft623) [inline composer.json:7]: Unrelated change!!!

Forward slashes are the standard for autoload paths in `composer.json`, regardless of the operating system. Using backslashes in path values is non-standard and breaks on Unix-like systems.

See examples on [Official Composer Documentation](https://getcomposer.org/doc/04-schema.md#psr-4).

Comment (SOF3) [reply composer.json:7]: pretty sure I was using Linux back then, no idea why I even wrote backslashes here...
